admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -466,6 +467,84 @@ In migrated Nextcloud installations the old behavior still applies, which is usi
 
   sudo -u www-data php occ config:app:set user_ldap enforce_home_folder_naming_rule --value=1
 
+"$home" Placeholder Field:
+  $home in an external storage configuration will be replaced with the value of the specified attribute
+
+.. _LDAP_User_Profile_Attributes:
+
+User Profile attributes
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: ../images/ldap-advanced-4-attributes.png
+   :alt: User Profile Attributes.
+
+After configuring those attributes, the User Profile data will be overwritten with the according data from LDAP.
+The data must be formatted like Nextcloud wants it, otherwise it will be constantly changed and rechanged/removed again.
+
+:Please be aware:
+- The user can not change the data in profile, for it will get overwritten almost instantly
+- The user can change the visibility scope in profile
+- The default visibility can be adjusted with setting the ``account_manager.default_property_scope`` array in ``config.php``
+- If multiple attribute values are present, only the first value is used
+- All user profile properties are limited to 2048 character
+- Having misformatted data in LDAP will most probably leave you with empty user profile fields
+- Setting the global ``profile.enabled => false`` on ``config.php`` skips the code
+
+By calling ``php occ ldap:check-user --update <uid>`` the users data from LDAP will be displayed and the profile gets updated.
+
+.. note:: After unsetting or removing an profile attribute, the data wont be reset or deleted from user profile.
+
+Phone Field:
+  The LDAP Attribute holding the phone number, to copy to the Profile Phone field.
+  The phone number has to be formatted in international syntax without delimiters (E.164).
+  Be sure to format phone numbers like ``+4966612345678``.
+
+  * Example: *telephoneNumber*
+  * Example: *mobile*
+
+.. note:: You should set your ``default_phone_region`` in ``config.php``.
+
+Website Field:
+  The LDAP attribute holding the website URI.
+  The URI must start with ``https://`` or ``http://`` others are currently not allowed in Nextcloud user profile.
+  If using ``labeledURI`` attributes the label (everything after first SPACE) gets removed.
+
+  * Example: *wWWHomePage*
+  * Example: *labeledURI*
+
+Address Field:
+  The LDAP attribute holding the users address. Location field on user profile page.
+  Nextcloud wants a single line value like ``city, country`` or ``somewhere under the loving sun``.
+  Multi line postalAddress format will get reformatted, ``$`` delimiter gets replaced with ``, ``.
+
+  * Example: *postalAddress*
+  * Example: *localityName*
+
+Twitter Field:
+  The LDAP attribute holding the Twitter account name.
```

Comment:
With or without the `@`?

Reply:
> With or without the `@`?

Actually Nextcloud doesn't bother or sanitize.
I think it should always be with, because for Twitter the `@` is defined as part of the handle `@username`
Same for Fediverse where the account name is defined to be `@name@instance`